tests/UI/pages/FO/FObasePage.ts

```diff
@@ -650,10 +650,11 @@ export default class FOBasePage extends CommonPage {
    * @return {Promise<Array<string>>}
    */
   async getFooterLinksTextContent(page: Page, position: number): Promise<Array<string>> {
-    return page.$$eval(
-      this.wrapperSubmenuItemLink(position),
-      (all) => all.map((el) => (el.textContent ?? '').trim()),
-    );
+    return page
+      .locator(this.wrapperSubmenuItemLink(position))
+      .evaluateAll(
+        (all) => all.map((el) => (el.textContent ?? '').trim()),
+      );
```

Comment:
```suggestion
      .locator(this.wrapperSubmenuItemLink(position))
      .allTextContents();
```

Reply:
done

Reply:
I didn't see your last version so i did not add the trim, doesn't work. so fixed now